### PR TITLE
Update WFS&C associations and rename wfs_combine to match workflow convention

### LINKED ISSUE #2672: Update WFS&C rules with actual example pools
# Issue
With the Wavefront Rehearsals,, there are solid examples now of coarse phasing. Update to the `Asn_WFSCMB` rule with how the dither pairs are actually delineated.

# Resources
- Proposal jw00747
- Proposal jw00711
- WFR 2
- #1340
- #1615

### PR DESCRIPTION
**Pull request by stscieisenhamer**
Resolves #2672 

Note to reviewers: code files are:
- jwst/associations/lib/rules_level3.py
- jwst/associations/lib/rules_level3_base.py
- jwst/pipeline/calwebb_wfs-image3.cfg
- jwst/stpipe/step.py
- jwst/wfs_combine/wfs_combine_step.py

### CODE REVIEW

**Comment by dmggh** [comment]
Does this have any bearing on the WCS issue in #2817 ?

File: jwst/wfs_combine/wfs_combine_step.py [hunk 1]
```diff
@@ -37,7 +37,7 @@ def process(self, input_table):
             output_model = wfs.do_all()
             output_model.meta.cal_step.wfs_combine = 'COMPLETE'
             self.save_model(
-                output_model, suffix='wfscmb', output_file=outfile
+                output_model, suffix='wfscmb', output_file=outfile, format=False
```
Comment (hbushouse) [inline jwst/wfs_combine/wfs_combine_step.py]: Does the step module need to be explicitly saving the output model, as opposed to just returning the model and letting stpipe handle it?

**Comment by stscieisenhamer** [comment]
Woah, some type of nasty build issue failure here. @rendinam

**Comment by nden** [comment]
@stscieisenhamer This is a known issue. Numpy 1.5.3 has a bug in masked arrays and astropy is pinned not to use it. Unfortunately it is the default version in conda now. Numpy 1.5.4 will soon become available in conda and this will be solved in a natural way. #2830 attempts to fix the np version in testing but for some reason only works in Travis.

**Comment by nden** [comment]
Included in the `0.12.2` patch.

**Pull request merged**